mdtraj/geometry/order.py

```diff
@@ -0,0 +1,263 @@
+##############################################################################
+# MDTraj: A Python Library for Loading, Saving, and Manipulating
+#         Molecular Dynamics Trajectories.
+# Copyright 2012-2015 Stanford University and the Authors
+#
+# Authors: Christoph Klein
+# Contributors:
+#
+# MDTraj is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 2.1
+# of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with MDTraj. If not, see <http://www.gnu.org/licenses/>.
+##############################################################################
+
+from __future__ import print_function, division
+
+import numpy as np
+NP18 = float(np.__version__[:3]) >= 1.8
+
+from mdtraj.geometry.distance import compute_center_of_mass
+from mdtraj.utils import ensure_type
+from mdtraj.utils.six import string_types
+
+
+__all__ = ['compute_nematic_order', 'compute_inertia_tensor']
+
+
+def compute_nematic_order(traj, indices='chains'):
+    """Compute the nematic ordering of a group in every frame.
+
```

Comment:
Can we add a one sentence description of the meaning of the nematic order parameter, for people (like me) that are/were not familiar with it? (e.g. 'the nematic order parameter measures the tendency of molecular groups to be oriented along a particular spatial direction', if that is an accurate characterization, or something similar).